packages/lit-element/src/lib/updating-element.ts

```diff
@@ -877,13 +767,13 @@ export abstract class UpdatingElement extends HTMLElement {
    * This method should be overridden instead. For example:
    *
    *   class MyElement extends LitElement {
-   *     async _getUpdateComplete() {
-   *       await super._getUpdateComplete();
+   *     async getUpdateComplete() {
+   *       await super.getUpdateComplete();
    *       await this._myChild.updateComplete;
    *     }
    *   }
    */
-  protected _getUpdateComplete() {
+  protected getUpdateComplete() {
```

Comment:
Are we sure we don't want to convert `updateComplete()` into a method?

We could use @kevinpschaaf's thenable hack:

```ts
if (DEV_MODE) {
  UpdatingElement.prototype.updateComplete.then = () => {
    throw new Error('updateComplete is a method');
  };
}
```

Reply:
https://github.com/Polymer/lit-html/issues/1253